jep/223/README.adoc

```diff
@@ -86,8 +86,44 @@ The changes to the existing permissions are detailed in the diagram below.
 
 image::Targetted_permission_model.png[Design of the new permission model.]
 
+Specifically, for an installation of Jenkins v2.207 with the default suggested plugins, the following configuration items would be available to a user with the `Overall/Manage` permission:
```

Comment:
This should make it clear that some of these are directly in scope for this JEP (the core stuff), while others are forward-looking and depend on maintainers' agreeing to use the new permission.